### PR TITLE
Re-organize project structure and implement AI Factory Agent Skills

### PR DESCRIPTION
**Pull request by slpatigdas**
This PR implements a major repository reorganization to follow a simplified folder structure and introduces a new set of specialized Agent Skills to automate repository management and planning tasks.

🗂️ Repository Reorganization
Reorganized the root directory into a Kriasoft-inspired structure to improve scannability and project maintenance:
- `/docs/`: Centralized task overviews, research, pricing, and usage guides.
- `/diagrams/`: Moved Mermaid charts and visual assets.
- `/src/`: Relocated source code and utility scripts (e.g., `scraper.py`).
- `/ideas/`: Dedicated space for brainstorming and future feature requests.
- `/shawn/`: Isolated personal scratchpads and task lists.
- `README.md`: Updated the "Steer File" with the new project structure and corrected all internal documentation links.

🛠️ New Agent Skills
Implemented five specialized `.md` skills in `.github/skills/` to provide the AI with domain-specific workflows:
1. File Architect: Enforces and maintains the new folder structure.
2. GitHub Manager: Automates descriptive commit messages and PR drafting.
3. Mermaid Architect: Translates raw requirements into system diagrams.
4. Tool Advisor: Recommends optimal AI models and orchestration patterns.
5. Vibe Testing: Stress-tests requirements for edge cases before coding.

Changes Included
- Moved: 14+ files to their new categorized directories.
- Created: `docs/File architecture update.md` (Migration log).
- Created: 5 Agent Skill definitions in `.github/skills/`.
- Updated: `README.md` with navigation and structure documentation.

### CODE REVIEW

File: .github/skills/mermaid-architect.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# 🧜‍♀️ Mermaid Architect
+
+## Description
+Translates raw project notes, folder structures, and system requirements into clean, stakeholder-friendly Mermaid diagrams. Designed specifically to visualize AI Factory frameworks, system architectures, and task flows to secure granular requirements from non-technical team members.
+
```
[the remark this answers is not in the document]
Comment (slpatigdas) [reply .github/skills/mermaid-architect.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/mindyoumh/my-ailounge/pull/10#discussion_r2887497829)

File: .github/skills/file-architect.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# 🗂️ File Architect
+
+## Description
+Maintains, enforces, and organizes the project's folder structure based on simplified Kriasoft conventions. Ensures all code, documentation, and visual assets are placed in their correct directories, and keeps the "steer file" (README.md) perfectly synced with the current architecture.
+
```
[the remark this answers is not in the document]
Comment (slpatigdas) [reply .github/skills/file-architect.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/mindyoumh/my-ailounge/pull/10#discussion_r2887497902)
